samples/confluent-schema-registry-integration/confluent-schema-registry-integration-producer2/pom.xml

```diff
@@ -15,23 +15,19 @@
 	</parent>
 
     <dependencies>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-actuator</artifactId>
-        </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.apache.avro</groupId>
-            <artifactId>avro</artifactId>
-            <version>${avro.version}</version>
-        </dependency>
         <dependency>
```

Comment:
Should we move all of the common deps up to the parent confluent-schema-registry-integration/pom.xml for these 3 apps? Or do we have them duplicated in here so peepz can easily see what each app has/requires? 

I had moved the avro deps up to that middle `avro-samples/pom.xml` in the original PR for the reason of de-duplicating. I am ok either way, just would like to know if its intentional or not.

Reply:
I think having it at a common location is fine. No need to duplicate. It was probably again an oversight to have them all over the place.